app/javascript/entrypoints/application.js

```diff
@@ -12,24 +16,4 @@ Turbo.start();
 // Example: Import a stylesheet in app/frontend/index.css
 import "../../assets/stylesheets/application.tailwind.css";
 
-addEventListener("turbo:before-render", (event) => {
-  if (document.startViewTransition) {
-    event.preventDefault();
-
-    document.startViewTransition(() => {
-      event.detail.resume();
-    });
-  }
-});
-
-addEventListener("turbo:before-frame-render", (event) => {
-  if (document.startViewTransition) {
-    event.preventDefault();
-
-    document.startViewTransition(() => {
-      event.detail.resume();
-    });
-  }
-});
```

Comment:
There is a tiny regression by removing those lines. They were used to transition the frame navigation. The best example is when you perform a search the results are displayed in a frame and with a subtle fade transition.

Can we get the same thing with Turn?

Reply:
hum very strange for Meilisearch you should have a folder `data.ms` can you try 

- stop all Meillisearch process 
- deleting this folder
- start the app again
- rails c
- Talk.reindex!

Reply:
And yes if you can revert this change from the test I did locally it does work with your changes

Reply:
I've not used meilisearch before. I installed it via brew, but the meilisearch-rails download/run link 404s so I gave up. (I'm not using docker.)

Reply:
I am not using docker neither. I think I used the cURL or brew. Is the Meillisearch process running on port 7700?

Reply:
No, if I run `meilisearch`, it just errors with the above message

Reply:
It's weird. I've never used it before, so no idea why I'm getting an upgrade error